File: implementation/project/Passenger.java
public class Passenger {
    private int id;
    private String name;
    private String phone;
    private String email;

    public Passenger(int id,String name,String phone,String email){
        this.id=id;
        this.name=name;
        this.phone=phone;
        this.email=email;
    }

    public void setId(int id){
        this.id=id;
    }
    public int getId(){
        return id;
    }

    public void setName(String name){
        this.name=name;
    }
    public String getName(){
        return name;
    }

    public void setPhone(String phone){
        this.phone=phone;
    }
    public String getPhone(){
        return phone;
    }

    public void setEmail(String email){
        this.email=email;
    }
    public String getEmail(){
        return email;
    }
    public void view (){
        System.out.println("id:"+id);
        System.out.println("name:"+name);
        System.out.println("phone:"+phone);
        System.out.println("email:"+email);
    }
}

File: implementation/Py-project/Py-project/Passenger.py
class Passenger:
  def __init__(self,Id, name,phone,email):
    self.__Id=Id
    self.__name = name
    self.__phone = phone
    self.__email = email

  def setName(self ,name):
      self.__name=name

  def getName(self):
     return self.__name

  def setPhone(self ,phone):
      self.__phone

  def getPhone(self):
     return self.__phone

  def getId(self):
     return self.__Id

  def setId(self ,Id):
      self.__Id

  def getEmail(self):
     return self.__email

  def setEmail(self ,email):
      self.__email

  def view(self):
    print("name:",self.__name)
    print("phone:",self.__phone)
    print("email:",self.__email)  